net.lutris.Lutris.yml

```diff
@@ -0,0 +1,366 @@
+            sources:
+              - type: archive
+                url: "https://github.com/kjd/idna/archive/v2.8.tar.gz"
+                sha256: db438aeba52c606cf1dd9671cb746377b4baeaea923397152e91576e8404d87a
+
+      - name: Pillow
+        buildsystem: simple
+        build-commands:
+          - python3 setup.py build -j $FLATPAK_BUILDER_N_JOBS
+          - python3 setup.py install --prefix=/app --root=/
+        sources:
+          - type: archive
+            url: "https://github.com/python-pillow/Pillow/archive/5.4.1.tar.gz"
+            sha256: d233c2e63414535281c572d2b4debd7de8a205b341ffc7db0f48f3f4d199fbc0
+
+  - name: lutris-app-environment
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /app/lib/i386-linux-gnu
+
+  - name: hwdata
+    config-opts:
+      - --datarootdir=/app/share
+    sources:
+      - type: archive
+        url: "https://github.com/vcrhonek/hwdata/archive/v0.322.tar.gz"
+        sha256: e71b66971a50f9aca5e7f1f2800e37d9015c69f03e68fd69fd1b687d1d1ba508
+
+  - name: pciutils
+    no-autogen: true
+    make-args:
+      - SHAREDIR=/app/share/hwdata
+      - OPT=-O2 -g
+    make-install-args:
+      - PREFIX=/app
+      - SBINDIR=/app/bin
+      - SHAREDIR=/app/share/hwdata
+      - MANDIR=/app/share/man
+    sources:
+      - type: archive
+        url: "https://mirrors.edge.kernel.org/pub/software/utils/pciutils/pciutils-3.6.2.tar.gz"
+        sha256: d5f9254f27bbda8243b345633e980144e6bd2af9c786bb8a152b904530aef599
+
+  - name: usbutils
+    config-opts:
+      - --datadir=/app/share/hwdata
+      - --sbindir=/app/bin
+    sources:
+      - type: archive
+        url: "https://mirrors.edge.kernel.org/pub/linux/utils/usb/usbutils/usbutils-007.tar.xz"
+        sha256: 7593a01724bbc0fd9fe48e62bc721ceb61c76654f1d7b231b3c65f6dfbbaefa4
+    modules:
+
+      - name: libusb
+        config-opts:
+          - --disable-udev
+        sources:
+          - type: archive
+            url: "https://github.com//libusb/libusb/releases/download/v1.0.22/libusb-1.0.22.tar.bz2"
+            sha256: 75aeb9d59a4fdb800d329a545c2e6799f732362193b465ea198f2aa275518157
+
+  - name: xrandr
+    sources:
+      - type: archive
+        url: "https://xorg.freedesktop.org/archive/individual/app/xrandr-1.5.0.tar.bz2"
+        sha256: c1cfd4e1d4d708c031d60801e527abc9b6d34b85f2ffa2cadd21f75ff38151cd
+
+  - name: mesa-demos
+    config-opts:
+      - --without-glut
+      - --bindir=/app/lib/mesa-demos
+    make-args:
+      - -C
+      - src/xdemos
+      - glxinfo
+    no-make-install: true
+    build-commands:
+      - install -D src/xdemos/glxinfo -t /app/bin/
+    sources:
+      - type: archive
+        url: "https://mesa.freedesktop.org/archive/demos/8.3.0/mesa-demos-8.3.0.tar.bz2"
+        sha256: c173154bbd0d5fb53d732471984def42fb1b14ac85fcb834138fb9518b3e0bef
+    cleanup:
+      - /lib/mesa-demos
+    modules:
+      - shared-modules/glew/glew.json
+      - shared-modules/glu/glu-9.0.0.json
```

Comment:
```suggestion
      - shared-modules/glu/glu-9.0.0.json
      - shared-modules/SDL/SDL-1.2.15.json
```

Reply:
Here shouldn't be extra indentation level, glu is a dependency of mesa-demos, while sdl is not.
And, better, [open a PR](/RobLoach/net.lutris.Lutris/pulls).

Reply:
Oops, you're right.

Reply:
Anyway, I think it's fine to leave this unfixed for now. Most of the native games available via Lutris are also available on Flathub and and Lutris is primarily used for Wine/emulated games, not the native ones.

Reply:
Also, the best way to fix this would, in my opinion, be adding the missing libs to the [Lutris runtime](https://github.com/lutris/lutris/wiki/Lutris-Runtime) instead of including them in a Flatpak.